src/modules/bootloader/main.py

```diff
@@ -193,9 +193,11 @@ def install_grub(efi_directory, fw_type):
     """
     if fw_type == "efi":
         print("Bootloader: grub (efi)")
+        install_path = libcalamares.globalstorage.value("rootMountPoint")
+        install_efi_directory = install_path + efi_directory
 
-        if not os.path.isdir(efi_directory):
-            check_target_env_call(["mkdir", "-p", "{!s}".format(efi_directory)])
+        if not os.path.isdir(install_efi_directory):
+            subprocess.call(["mkdir", "-p", "{!s}".format(install_efi_directory)])
```

Comment:
Could you explain this change? To me it seems like check_target_env_call(efi_directory) and this subprocess.call would do the exact same thing?